src/qt/bitcoingui.cpp

```diff
@@ -104,7 +108,7 @@ BitcoinGUI::BitcoinGUI(const PlatformStyle *platformStyle, const NetworkStyle *n
 {
     GUIUtil::restoreWindowGeometry("nWindow", QSize(850, 550), this);
 
-    QString windowTitle = tr("Bitcoin Core") + " - ";
+    QString windowTitle = tr(PACKAGE_NAME) + " - ";
```

Comment:
Oh, right, does this work then? Does qtranslate pick up the string out of `tr(PACKAGE_NAME)`, even though it's a macro? Same for `_(PACKAGE_NAME)` in gettext?